### DISCUSSION TITLE
Did I kill the USB port?

### DISCUSSION THREAD

**Discussion by iz3man** [Louisvdw/dbus-serialbattery#124] [Q&A] [answered]
I tried several FTDI adapaters and finally had my BMS connected. But I also connected the VCC of the BMS to the serial adapter, as I didn't read the warning.
Today the adapter is not recognized, and NO device I connect to the multiplus seems to be seen.

`May 15 08:54:06 nanopi user.err kernel: [  134.595776] usb usb3-port1: Cannot enable. Maybe the USB cable is bad?
May 15 08:54:06 nanopi user.info kernel: [  134.602410] usb usb3-port1: attempt power cycle
May 15 08:54:08 nanopi user.err kernel: [  136.205767] usb usb3-port1: Cannot enable. Maybe the USB cable is bad?
May 15 08:54:09 nanopi user.err kernel: [  137.465810] usb usb3-port1: Cannot enable. Maybe the USB cable is bad?
May 15 08:54:09 nanopi user.err kernel: [  137.472438] usb usb3-port1: unable to enumerate USB device
May 15 08:54:28 nanopi daemon.warn connmand[605]: Online check failed for 0x574bc8 Wired
May 15 08:54:28 nanopi daemon.info connmand[605]: eth0 {del} route 82.165.8.211 gw 192.168.1.1 scope 0 <UNIVERSE>
May 15 08:58:51 nanopi user.err kernel: [  419.225806] usb usb2-port1: Cannot enable. Maybe the USB cable is bad?
May 15 08:58:53 nanopi user.err kernel: [  421.145808] usb usb2-port1: Cannot enable. Maybe the USB cable is bad?
May 15 08:58:53 nanopi user.info kernel: [  421.152430] usb usb2-port1: attempt power cycle
May 15 08:58:55 nanopi user.err kernel: [  423.415821] usb usb2-port1: Cannot enable. Maybe the USB cable is bad?
May 15 08:58:57 nanopi user.err kernel: [  425.335930] usb usb2-port1: Cannot enable. Maybe the USB cable is bad?
May 15 08:58:57 nanopi user.err kernel: [  425.342791] usb usb2-port1: unable to enumerate USB device
May 15 08:58:59 nanopi user.err kernel: [  426.875775] usb usb3-port1: Cannot enable. Maybe the USB cable is bad?
`

Did I destroy the USB port? And if so: Any idea what I can do?
Thanks!

**Comment by Louisvdw** [comment, marked as answer]
Hi @iz3man 

It is possible that if you connected a voltage to the USB port that is too high that you might have damaged some components. USB works on 5V so anything above that could be harful and some BMS output the full battery voltage on their Vcc would could be 50V or more.
 
I would expect that the USB adapter would be the item that will get damaged. Check if other adpters are picked up and also check a USB flash drive, but if nothing works then your Mutiplus might have some damage and you will have to send it in for repairs. Luckely that should not be a difficult repair, and Victron normally have very good support.

**Comment by iz3man** [comment]
Thanks @Louisvdw . Sorry for posting it in Issues, didn't notice the Discussion tab. Thanks for moving this!

The serial-adapter was set to 3.3V just to be sure. And there is only 3.3V and 5V. Nothing more, so no way any higher voltages could be there. USB Flashdrives aren't working either. I already filed an RMA with the seller and hopefully will receive a replacment soon.
